pkg/landscaper/installations/executions/template/testdata/gotemplate/component-descriptor-12.yaml

```diff
@@ -0,0 +1,212 @@
+component:
```

Comment:
can we make the example a bit shorter?
maybe only one example ressource is enough.

Reply:
we shortened the test data